site/_docs/configuration.md

```diff
@@ -201,6 +201,14 @@ class="flag">flags</code> (specified on the command-line) that control them.
         <p><code class="flag">--drafts</code></p>
       </td>
     </tr>
+        <tr class="setting">
+      <td>
+        <p class="name"><strong>Environment</strong></p>
+        <p class="description">Use a specific environment value in the build</p>
+      </td>
+      <td class="align-center">
+        <p><code class="flag">JEKYLL_ENV=production</code></p>
+      </td>
```

Comment:
Whoops! Looks like there is no closing `</tr>` here. Mind adding it and matching the indentation of the other lines?